homeassistant/components/netatmo/__init__.py

```diff
@@ -148,13 +160,30 @@ async def register_webhook(event):
             webhook_register(
                 hass, DOMAIN, "Netatmo", entry.data[CONF_WEBHOOK_ID], handle_webhook
             )
+
+            async def handle_event(event):
+                """Handle webhook events."""
+                if event["data"]["push_type"] == "webhook_activation":
+                    if sub is not None:
+                        _LOGGER.debug("sub called")
+                        sub()
+
+                    if unsub is not None:
+                        _LOGGER.debug("Unsub called")
+                        unsub()
+
+            sub = async_dispatcher_connect(
+                hass,
+                f"signal-{DOMAIN}-webhook-None",
```

Comment:
Why `None` in the signal name?

Reply:
It's the `event_type` (or the absence of) that is appended by the webhook handler.